packages/client.api/src/aggregate/AggregationsResults.test.ts

```diff
@@ -77,9 +79,11 @@ type Q = AggregationResultsWithoutGroups<
   }["objects"]["Todo"],
   {
     "locationCity:approximateDistinct": "unordered";
+    "locationCity:exactDistinct": "unordered";
     "id:max": "unordered";
     "id:sum": "unordered";
     "text:approximateDistinct": "unordered";
+    "text:Distinct": "unordered";
```

Comment:
I'm not fully sure if the tests in this file are doing anything, but I added them here just in case I was wrong and could be used for the future. Can we remove this file if its not being used?

Reply:
Yeah they maybe tested something at one time but clearly do nothing now.

Reply:
Yea, @nihalbhatnagar let's maybe do any file deletions as a FLUP to this PR, so we can keep this PR to just adding the new functionality.

Reply:
Cool I'll do that